episodes/severity-static.Rmd

```diff
@@ -0,0 +1,788 @@
+
+The estimator for CFR can be written as: 
+
+$$p_{t} = \frac{b_{t}}{u_{t}}$$
+
+where $p_{t}$ is the realized proportion of confirmed cases to die from the infection (or the unbiased CFR), and $b_{t}$, the crude and biased estimate of CFR (also naive CFR).
+
+From this last equation, we observe that the unbiased CFR $p_{t}$ is larger than biased CFR $b_{t}$ because in $u_{t}$ the numerator is smaller than the denominator (note that $f_{t}$ is the probability distribution of the *delay distribution* between onset and death). Therefore, we refer to $b_{t}$ as the biased estimator of CFR.
+
+When we observe the entire course of an epidemic (from $t \rightarrow \infty$), $u_{t}$ tends to 1, making $b_{t}$ tends to $p_{t}$ and become an unbiased estimator ([Nishiura et al., 2009](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0006852)).
+
+:::::::::::::::::::::::::::
+
+:::::::::::::::::: callout
+
+### Use the `<epidist>` class
+
+When using an `<epidist>` class object we can use this expression as a template:
+
+`function(x) density(<EPIDIST_OBJECT>, x)`
+
+For distribution functions with parameters not available in `{epiparameter}`, we suggest you two alternatives: 
+
+- Create an `<epidist>` class object, to plug into other R packages of the outbreak analytics pipeline. Read the [reference documentation of `epiparameter::epidist()`](https://epiverse-trace.github.io/epiparameter/reference/epidist.html), or
+
+- Read `{cfr}` vignette for [a primer on working with delay distributions](https://epiverse-trace.github.io/cfr/articles/delay_distributions.html).
+
+::::::::::::::::::
+
+:::::::::::::::::: spoiler
+
+## When to use discrete distributions?
+
+For  `cfr_static()` and all the `cfr_*()` family of functions, the most appropriate choice to pass are **discrete** distributions. This is because we will work with daily case and death data.
+
+We can assume that evaluating the Probability Distribution Function (PDF) of a *continuous* distribution is equivalent to the Probability Mass Function (PMF) of the equivalent *discrete* distribution.
+
+However, this assumption may not be appropriate for distributions with greater picks. For example, when onset-to-death is strongly peaked with a low variance. In such cases, the average disparity between the PDF and PMF is expected to be more pronounced compared to distributions with broader spreads.
+
+In a scenario like the last one, you can opt to discretise the continuous distribution using `epiparameter::discretise()` to an `<epidist>` object.
+
+::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::: challenge
+
+Load the `ebola_kikwit_1995` from `{outbreaks}` and estimate the delay-adjusted CFR using the appropriate distribution delay. Then:
+
+- Compare the naive and the delay-adjusted CFR solutions!
+
+:::::::::::::::::::: hint
+
+- Find the appropriate `<epidist>` object!
```

Comment:
what does this mean? in the solution you're using the same epidist object that was used for ebola1976

Reply:
just realized this! I think I assumed it was a different disease. I'll check if any other fits better. #42